src/models/entities/entity.ts

```diff
@@ -0,0 +1,7 @@
+export interface Entity {
```

Comment:
@madhav-devslane This interface should have a generic so that it can have the flexibility to support `string` and `number` identifiers. Like

```
export interface Entity<T = number> {
    id: T;
}
```